src/main/java/org/prebid/server/bidder/adquery/model/response/AdQueryDataResponse.java

```diff
@@ -0,0 +1,37 @@
+package org.prebid.server.bidder.adquery.model.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.List;
+
+@Builder(toBuilder = true)
+@Getter
+public class AdQueryDataResponse {
+
+    @JsonProperty("requestId")
+    String requestId;
+
+    @JsonProperty("creationId")
+    Integer creationId;
+
+    String currency;
+
+    String cpm;
+
+    String code;
+
+    @JsonProperty("adqLib")
+    String adqLib;
+
+    String tag;
+
+    @JsonProperty("adDomains")
+    List<String> adDomains;
+
+    String deadlid;
```

Comment:
typo: `dealid`, and since it's the deal id, I prefer using camel case and then JsonProperty will be required